python/eos/analysis.py

```diff
@@ -155,6 +155,18 @@ def __init__(self, priors, likelihood, global_options={}, manual_constraints={},
         for n in varied_parameter_names - used_parameter_names:
             eos.warn('likelihood does not depend on parameter \'{}\'; remove from prior or check options!'.format(n))
 
+    def __sanitize_manual_input(self, data):
```

Comment:
Why the double underscore here? Will there be derived classes with their own sanitisation method?

Reply:
Isn't that the syntax to write an internal function in a python class?

Reply:
A single underscore politely indicates that a function is not intended for the user api and double underscores are processed by the interpreter to be class-specific to avoid accidents. Everything would work, but I'd say a single underscore is the most conventional.